arch/inst/V/vsetivli.yaml

```diff
@@ -3,9 +3,11 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: vsetivli
-long_name: No synopsis available.
+long_name: Set vector configuration (immediate immediate).
```

Comment:
eww. :-)

Suggest "Vector Set Vector Type Immediate and Vector Length Immediate" (not great, I know)